### DISCUSSION TITLE
can i config static ip in web-server sketches for audio-tools?

### DISCUSSION THREAD

**Discussion by DragonZideutch** [pschatzmann/arduino-audio-tools#1753] [Q&A] [answered]
trying to find information on how to enable static ip adresses for audio-tools?

i2s - http server wav, etc….

thanks for your great help

**Comment by pschatzmann** [comment, marked as answer]
This is nothing to do with the AudioTools. 
Just google how to do this for your processor...

What did you try ?

**Comment by DragonZideutch** [comment]
thanks i will look into that, taught there was some predefine function in audiotools, but now i know where to look for that information. you are a wealth of infos.have a great day sir.DragonZideutchLe 24 oct. 2024 à 02:34, Phil Schatzmann ***@***.***> a écrit :﻿
This is nothing to do with the AudioTools.
Just google how to do this for your processor...
